source/includes/steps-configure-user-metadata-import-export.yaml

```
@@ -13,16 +13,19 @@ content: |
   .. tip::
      
      You can also download a copy of your application's configuration files from
-     the :guilabel:`Deploy > Export App` screen in the {+ui+}.
+     the {+ui+}. Go to the :guilabel:`Deploy > Export App` screen from
+     the Realm Application Dashboard.
 ---
 title: Configure User Metadata
 ref: configure-user-metadata
 content: |
-  To configure your application to read user metadata from the authentication
-  provider, update the :ref:`metadata_fields <config-auth-providers>` in
-  the ``/auth/providers.json``:
+  The authentication provider :ref:`metadata_fields <config-auth-providers>` 
+  for your app live in ``/auth/providers.json``. Update this object to request
```

metadata_fields isn't object, it's array

```suggestion
  for your app live in ``/auth/providers.json``. Update ``metadata_fields`` to request
```